simgui.json

```diff
@@ -0,0 +1,62 @@
+{
+  "NTProvider": {
+    "types": {
+      "/FMSInfo": "FMSInfo",
+      "/SmartDashboard/Autonomous Mode": "String Chooser",
+      "/SmartDashboard/Field": "Field2d"
+    },
+    "windows": {
+      "/SmartDashboard/Field": {
+        "bottom": 544,
+        "builtin": "2024 Crescendo",
+        "fused_pose": {
+          "color": [
+            0.0588231086730957,
+            0.0,
+            1.0,
+            255.0
+          ]
+        },
+        "height": 8.013679504394531,
+        "left": 46,
+        "right": 1088,
+        "top": 36,
+        "width": 16.541748046875,
```

Comment:
These field dimensions are wrong, notice the yellow border doesn't line up with the walls in the field image:

![screenshot](https://github.com/thedropbears/pycrescendo/assets/128854/77e3cb69-2d9b-44a6-ad39-6cb1d6e607f2)

Reply:
I have a patch that uses the newest image from WPILib, and sets the coordinates correctly.
The `.png` file is 2MB. Should I force commit it and add to this PR?

Reply:
I have pushed the correct dimensions